libgraph/include/katana/GraphTopology.h

```diff
@@ -1804,16 +1814,23 @@ class KATANA_EXPORT PGViewCache {
         pg, node_types, edge_types, *this);
   }
 
+  // Avoids a copy of the default topology.
+  const GraphTopology& GetDefaultTopologyRef() const noexcept;
+
+  void DropAllTopologies() noexcept;
+
 private:
-  std::shared_ptr<GraphTopology> GetOriginalTopology(
-      const PropertyGraph* pg) const noexcept;
+  std::shared_ptr<GraphTopology> GetDefaultTopology() const noexcept;
 
   std::shared_ptr<CondensedTypeIDMap> BuildOrGetEdgeTypeIndex(
       const PropertyGraph* pg) noexcept;
 
+  // The pop flag ensures that the returned topology is not
+  // in the edge_shuff_topos_ cache.
   std::shared_ptr<EdgeShuffleTopology> BuildOrGetEdgeShuffTopo(
       PropertyGraph* pg, const tsuba::RDGTopology::TransposeKind& tpose_kind,
-      const tsuba::RDGTopology::EdgeSortKind& sort_kind) noexcept;
+      const tsuba::RDGTopology::EdgeSortKind& sort_kind,
+      bool pop = false) noexcept;
```

Comment:
Nit: it is preferred a second function rather than use default arguments (second func can just call the other with the default argument)